Database-Features/removing-corrupt-data.md

```diff
@@ -77,7 +77,7 @@ set verbose on;
 @/tmp/enforce_indexes.sql;
 ```
 
-To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f recreate_indexes.sql`. You can view `/tmp/index_logs.log` or auditing in the database to ensure that all queries executed successfully.
+To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f recreate_indexes.sql`. Database user requires privileges to create and drop indices to run this script. DBA role would be sufficient, see [Roles](https://docs.exasol.com/db/latest/database_concepts/database_users_roles.htm#Roles). You can view `/tmp/index_logs.log` or auditing in the database to ensure that all queries executed successfully.
```

Comment:
For simplification, I would also put here that it requres DBA privileges, no need to mention something specific about creating and dropping indices